lib/models/note.js

```diff
@@ -584,5 +586,34 @@ module.exports = function (sequelize, DataTypes) {
     return operations
   }
 
+  Note.lockOldGuestNotes = async function (seconds, permission) {
```

Comment:
I think the parameter `seconds` should be called `secondsUntilLock` or something along those lines.